cmd/push.go

```diff
@@ -57,6 +55,7 @@ push -P -A (all accounts)`,
 	cmd.Flags().StringVarP(&params.removeAcc, "account-removal", "R", "", "remove specific account. Only works with nats-resolver enabled nats-server. Mutually exclusive of prune/diff.")
 	cmd.Flags().StringVarP(&params.sysAcc, "system-account", "", "", "System account for use with nats-resolver enabled nats-server. (Default is system account specified by operator)")
 	cmd.Flags().StringVarP(&params.sysAccUser, "system-user", "", "", "System account user for use with nats-resolver enabled nats-server. (Default to temporarily generated user)")
+	cmd.Flags().IntVarP(&params.timeout, "timeout", "", 1, "timeout in seconds [1-60] to wait for responses from the server (only applicable to nats-resolver configurations, and applies per operation)")
```

Comment:
Please add note about the default value in description

Reply:
```bash
$ nsc push -h
Push an account jwt to an Account JWT Server

Usage:
  nsc push (currentAccount)
push -a <accountName>
push -A (all accounts)
push -P
push -P -A (all accounts) [flags]

Examples:
push

Flags:
  -a, --account string                  account name
  -u, --account-jwt-server-url string   set account jwt server url for nsc sync (only http/https/nats urls supported if updating with nsc) If a nats url is provided 
  -R, --account-removal string          remove specific account. Only works with nats-resolver enabled nats-server. Mutually exclusive of prune/diff.
  -A, --all                             push all accounts under the current operator (exclusive of -a)
  -D, --diff                            diff accounts present in nsc env and nats-account-resolver. Mutually exclusive of account-removal/prune.
  -F, --force                           push regardless of validation issues
  -h, --help                            help for push
  -P, --prune                           prune all accounts not under the current operator. Only works with nats-resolver enabled nats-server. Mutually exclusive of account-removal/diff.
      --system-account string           System account for use with nats-resolver enabled nats-server. (Default is system account specified by operator)
      --system-user string              System account user for use with nats-resolver enabled nats-server. (Default to temporarily generated user)
      --timeout int                     timeout in seconds [1-60] to wait for responses from the server (only applicable to nats-resolver configurations, and applies per operation) (default 1)
```